docs/content/recipes/interoperability/identifier-mapping.md

```diff
@@ -0,0 +1,384 @@
+
+
+## Graphical Overview of the FAIRification Recipe Objectives
+
+This recipe will cover the highlighted topics
+```mermaid
+    graph TB
+      I[Identifier]
+      I --> IM[Identifier Mapping]
+      M[Mapping] --> OM[Ontology Mapping]
+      OM --> OxO
+
+      M --> IM
+
+      IM -- has section --> IMS
+
+      IM -- has section--> IE([Identifier Equivalence])
+      IE -- mentions--> SL{{Scientific Lenses}}
+
+      IMS([Identifier Mapping Services])
+      IMS -- mentions --> BDb[BridgeDb]
+      IMS -- mentions --> UniChem{{UniChem}}
+      IMS -- mentions --> Sameas{{Sameas}}
+      IMS -- mentions --> Identifiers{{Identifiers.org}}
+
+      BDb -- has section --> MGG[Mapping global identifiers to other global identifiers]
+      BDb -- has section --> MLG[Mapping local identifiers to other global identifiers]
+
+      MGG -- has section --> P1([Python])
+      MGG -- has section --> R1([R])
+      MLG -- has section --> P2([Python])
+      MLG -- has section --> R2([R])
+
+        style OM stroke-dasharray: 5 5
+        %% style M stroke-dasharray: 5 5
+
+       classDef select fill:#f9f;
+       class IM select;
+       class IMS select;
+       class BDb select;
+       class UniChem select;
+       class Sameas select;
+       class Identifiers select;
+       class M select;
+       class IE select;
+       class SL select;
+      %% style IM fill:#f9f
+      %% style I fill:#f9f
+      %% style OxO fill:#f9f
+```
+___
+
+**ToDo:** Update diagram
+
+
+<div class="mermaid">
+graph TD
+
+  Process2[Identifier Minting]:::bix --> A1(authority decide identity):::box
+  Process2 --> Process3
+  Process3[URI construction]:::bix --> A2(scoping the authority):::box
+  Process3 --> Process4
+  Process4[URI resolution]:::bix --> A3(directing requests to the relevant identified entity):::box
+
+  classDef bix font-family:avenir,font-size:14px,fill:#7e0038,stroke:#222,color:#fff,stroke-width:1px
+  classDef box font-family:avenir,font-size:14px,fill:#2a9fc9,stroke:#222,color:#fff,stroke-width:1px
+</div>
+
+---
+
+
+## Capability & Maturity Table
+
+| Capability  | Initial Maturity Level | Final Maturity Level  |
+| :------------- | :------------- | :------------- |
+| Interoperability | minimal | repeatable |
+| Identifier mapping ||repeatable|
+
+----
+<!-- 
+## FAIRification Objectives, Inputs and Outputs
+
+| Actions.Objectives.Tasks  | Input | Output  |
+| :------------- | :------------- | :------------- |
+| [Identifier](http://edamontology.org/data_0842) |   |   |
+ -->
+
+## Table of Data Standards
+
+| Data Formats  | Terminologies | Models  |
+| :------------- | :------------- | :------------- |
+| [IRI](https://tools.ietf.org/html/rfc3987) |   |   |
+| [CURIE](https://www.w3.org/TR/2010/NOTE-curie-20101216/) |   |   |
+| [URL](https://tools.ietf.org/html/rfc1738) |  |  |
+| [HTTP]() |  |  |
+| [RDF]() |  |  |
+| [CSV]() |  |  |
+| [TSV]() |  |  |
```

Comment:
Four missing links.

Reply:
Added some links, not sure if the correct ones, @AlasdairGray can check